appveyor.yml

```diff
@@ -49,10 +49,11 @@ install:
     - IF %PHP%==1 echo extension=php_pdo_sqlite.dll >> php.ini-max
     - IF %PHP%==1 echo extension=php_curl.dll >> php.ini-max
     - IF %PHP%==1 echo curl.cainfo=c:\php\cacert.pem >> php.ini-max
-    - appveyor DownloadFile https://getcomposer.org/composer.phar
+    - appveyor DownloadFile https://getcomposer.org/download/1.0.0/composer.phar
```

Comment:
That's why we run `composer self-update` after downloading 1.0.0 (I don't think that we want to update this file with every new Composer release).